modules/launch_utils.py

```diff
@@ -352,6 +372,8 @@ def prepare_environment():
         run(f'"{python}" -m {torch_command}', "Installing torch and torchvision", "Couldn't install torch", live=True)
         startup_timer.record("install torch")
 
+    if args.use_ipex:
+        args.skip_torch_cuda_test = True

```

Comment:
would be good to include a torch version check, if users have torch-cu118 installed then run pip install to install required packages

Reply:
```python

if args.use_ipex:
    if is_installed("torch"):
        import torch
        if torch.__version__ != "2.0.0a0+git9ebda2" or not is_installed("intel_extension_for_pytorch"):
            run(f'"{python}" -m {torch_command}', "Installing torch and torchvision", "Couldn't install torch", live=True)
            startup_timer.record("install torch")

```

Reply:
Or we could `check_run_python("import torch; import intel_extension_for_pytorch; assert torch.xpu.is_available()")` to perform a sanity test, so that we don't assume a specific torch version -- Intel may release newer versions and user could build from source with a custom version.